includes/polling/os/unix.inc.php

```diff
@@ -43,9 +43,30 @@
         $hardware = "Dell " . $hw;
     }
 
+    # Try detect using the extended option (dmidecode)
+    if (!$hw) {
+        $version_dmide = str_replace("\"", "", snmp_get($device, ".1.3.6.1.4.1.2021.7890.2.4.1.2.8.104.97.114.100.119.97.114.101.1", "-Oqv", "NET-SNMP-EXTEND-MIB"));
```

Comment:
You've specified the MIB to use but have a numeric OID, use the named OID instead.